app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.AppCompat.Light.NoActionBar">
+        <activity android:name=".activity.ReservationActivity"></activity>
```

Comment:
here you can simplify, doing this
 `<activityandroid:name=".activity.ReservationActivity"/>`

Reply:
Done